src/lib/run-studies.ts

```diff
@@ -138,7 +141,30 @@ const main = async (): Promise<void> => {
             run.containerLocation,
             run.title,
         )
-    })
+    }
+
+    await deleteECSTaskDefinitions(
+        ecsClient,
+        filterOrphanTaskDefinitions(
+            result,
+            (await getTaskDefinitionsWithRunId(taggingClient)).ResourceTagMappingList || [],
```

Comment:
small nitpick, maybe just pull this out as a variable onto line 145 for legibility. could potentially pull out the `filterOrphanTaskDefinitions` call as well 🤔

Reply:
Yeah, I felt like I was getting greedy keeping this to one line :sweat_smile: , so thanks for the sanity check there. Updated based on your suggestion in [this commit](https://github.com/safeinsights/setup-app/pull/10/commits/1a0deb0807199a72bbb87810542b23805770742d).